cpmpy/expressions/__init__.py

```diff
@@ -20,6 +20,6 @@
 # others need to be imported by the developer explicitely
 from .variables import boolvar, intvar, cpm_array
 from .variables import BoolVar, IntVar, cparray # Old, to be deprecated
-from .globalconstraints import AllDifferent, AllEqual, Circuit, Table, Minimum, Maximum, Element
+from .globalconstraints import AllDifferent, AllEqual, Circuit, Table, Minimum, Maximum, Element, XOR
```

Comment:
We used single capital alwasy for now, e.g. Xor()... any strong reason to want to full-cap here?

Reply:
No special reason. I'll change it to make it coherent with other global constraints